src/full/Agda/Syntax/Concrete/Operators.hs

```diff
@@ -671,6 +675,21 @@ classifyPattern :: PatternCheckConfig -> Pattern -> PM ParseLHS
 classifyPattern conf p =
   case patternAppView p of
 
+    -- case @... | ps@
+    Arg _ (Named _ (EllipsisP _ mp)) : ps -> do
+      let wps = map (unWithP . namedArg) ps
+      mapM_ valid wps
+      classifyPattern conf (fromMaybe __IMPOSSIBLE__ mp) >>= \case
+        ParseLHS f core -> return $ ParseLHS f $ LHSWith core True wps []
+        ParsePattern p  ->
```

Comment:
I am ending up in case `ParsePattern` when I expect to always get a `ParseLHS` for the content of an `EllipsisP`.